model/src/main/kotlin/config/PackageConfigurationProviderConfiguration.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+import org.ossreviewtoolkit.utils.common.Plugin
+
+data class PackageConfigurationProviderConfiguration(
```

Comment:
Please also document the class itself.

Reply:
Can we reuse / generalize `PackageCurationProviderConfiguration` here?

Reply:
I believe that this falls under the category of too early refactoring, because I believe there will be provider specific configuration. E.g. we discussed already for curations, that one may be interested in consuming curations only of a certain type. Also, that is the benefit of using same class for different purposes? (If there should be code re-use potential, re-use cannot be achieved also with separate classes)

Reply:
I guess that question boils down to whether provider-specific (not: plugin-specific) configuration should go to the existing `config` map, or whether a dedicated property should be introduced.

Reply:
> I guess that question boils down to whether provider-specific (not: plugin-specific) configuration should go to the existing `config` map, or whether a dedicated property should be introduced.

Oh right, I missed that. So, then I'm fine with just staying with the current approach (no dedicated properties).